plugins/catalog-backend/package.json

```diff
@@ -21,6 +21,7 @@
     "helmet": "^3.22.0",
     "knex": "^0.21.1",
     "morgan": "^1.10.0",
+    "sqlite3": "^4.2.0",
```

Comment:
Not quite sure why it wasn't there and how it worked for you @freben :)

Reply:
yeah that's indeed weird! probably pulled in as transient dependency somehow, not sure